otter_buddy/cogs/interview_match.py

```diff
@@ -65,8 +67,17 @@ async def send_weekly_message(self):
                 entry["message_id"] = message.id
                 db_interview_match.DbInterviewMatch.set_interview_match(entry)
     
-    async def check_weekly_message(self):
-        weekday = (datetime.datetime.today().weekday() - 1 + 7) % 7
+    async def check_weekly_message(self, weekday = None) -> None:
+        weekday = (datetime.datetime.today().weekday() - 1 + 7) % 7 if weekday is None else weekday
+        """
+        Above operation:
```

Comment:
Can you put the comment before the action (you can remove the "above operation"? Is better for reading it.

Reply:
done